scripts/zones/AlTaieu/mobs/Ul_hpemde.lua

```diff
@@ -24,6 +28,55 @@ end;
 -----------------------------------
 
 function onMobEngaged(mob, killer)
+    mob:hideName(false);
+    mob:untargetable(false);
+    if (mob:AnimationSub() == 5) then
+        mob:AnimationSub(6);
+    end
+end;
+
+-----------------------------------
+-- onMobRoam Action
+-----------------------------------
+
+function onMobRoam(mob)    
+    if (mob:getHPP() == 100) then 
+        mob:setLocalVar("damaged", 0);
+        mob:SetAutoAttackEnabled(false);
+        mob:SetMobAbilityEnabled(false);
+    end
+end;
+
+-----------------------------------
+-- onMobFight
+-----------------------------------
+
+function onMobFight(mob, target)
+
+    if (mob:getHPP() < 100) then -- it will attack once it has been damaged.
```

Comment:
not that i checked whether getHPP returns float or int, but its prolly best to check hp < max hp instead of hp% (cause 99.99% would still come out as 100% - probably)